guides/specification.md

```diff
@@ -0,0 +1,555 @@
+# Trento Checks DSL Specification
+
+A language allowing to declare best practices to be adhered on target SAP Infrastructures.
+
+# Table of contents
+
+- [Introduction](#introduction)
+- [Anatomy of a Check](#anatomy-of-a-check)
+  - [Filename Convention](#filename-convention)
+  - [Structure](#structure)
+    - [id](#id)
+    - [name](#name)
+    - [group](#group)
+    - [description](#description)
+    - [remediation](#remediation)
+    - [severity](#severity)
+    - [Facts, Values, Expectations](#facts-values-expectations)
+- [Facts](#facts)
+- [Values](#values)
+- [Expectations](#expectations)
+- [Expression Language](#expression-language)
+    - [Evaluation Scope](#evaluation-scope)
+
+## Introduction
+
+The need this DSL aims to fulfill is to provide users a simple way to declare what we (the Trento Core Team) often refer to as "Checks". 
```

Comment:
Do we need to reference the `Trento Core Team` here?
I would refer to `Trento Team` in any case